changelog/issue-6245.md

```diff
@@ -0,0 +1,14 @@
+audience: users
+level: minor
+reference: issue 6245
+---
+Generic Worker payload now supports declaratively mounting indexed artifacts into the task directory. For example:
```

Comment:
is it worth adding that workers would need an extra scope to be able to "talk to index artifacts"?